From f46213f92bdedb41fa3d536a484b54c34b5bc275 Mon Sep 17 00:00:00 2001
From: kopyt <ptrules4ever@gmail.com>
Date: Mon, 18 Mar 2019 10:46:15 +0100
Subject: [PATCH] imx-boot workaround for dtc unknown in mkimage

---
 iMX8M/soc.mak | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/iMX8M/soc.mak b/iMX8M/soc.mak
index 1b6e357..2fc99ef 100644
--- a/iMX8M/soc.mak
+++ b/iMX8M/soc.mak
@@ -1,6 +1,8 @@
 MKIMG = mkimage_imx8
 OUTIMG = flash.bin
 
+PATH := /usr/bin:/bin:$(PATH)
+
 CC ?= gcc
 CFLAGS ?= -O2 -Wall -std=c99 -static
 INCLUDE = ./lib
-- 
2.20.1

